torch/onnx/utils.py

```diff
@@ -483,8 +483,20 @@ def _model_to_graph(model, args, verbose=False,
                             module=module)
     from torch.onnx.symbolic_helper import _onnx_shape_inference
     if isinstance(model, torch.jit.ScriptModule) or isinstance(model, torch.jit.ScriptFunction):
-        assert example_outputs is not None, "example_outputs must be provided when exporting a ScriptModule or " \
-                                            "ScriptFunction."
+        input_args = copy.deepcopy(args)
+        input_kwargs = {}
+        if input_args and isinstance(input_args[-1], dict):
+            input_kwargs = input_args[-1]
```

Comment:
There is same logic in `_decide_input_format` to compute `input_kwargs` and `input_args`, can we use a function to reuse the code?

Reply:
Thanks for this comment.
The `_decide_input_format` is used before calling `_model_to_graph()`.https://github.com/pytorch/pytorch/blob/66846701b85715747090d79c1c3d793731747328/torch/onnx/utils.py#L702 for decideing the format of input.
In this part, we only need get the kwargs and args so we couldn't reuse that.